src/js/types/generic.type.ts

```diff
@@ -20,6 +18,10 @@ export type Tooltip = GenericObject;
 
 export type Localization = GenericObject;
 
+export type DrmConfig = GenericObject;
```

Comment:
when to type this sucker?

Reply:
is now the time?

Reply:
this should be a pass through we define in commercial

Reply:
cool, will leave here until the TS commercial effort kicks off